hummingbot/client/command/gateway_command.py

```diff
@@ -50,6 +51,96 @@ def gateway(self,
             safe_ensure_future(self._generate_certs())
         elif option == "test-connection":
             safe_ensure_future(self._test_connection())
+        elif option == "start":
+            safe_ensure_future(self._start_gateway())
+        elif option == "stop":
+            safe_ensure_future(self._stop_gateway())
+
+    @staticmethod
+    def get_gateway_container_name() -> str:
+        instance_id_suffix: str = global_config_map["instance_id"].value[:8]
+        return f"hummingbot-gateway-{instance_id_suffix}"
```

Comment:
Is there a reason for redefining this?

Reply:
Yea I must have included that by mistake, I will remove it accordingly.